### PR TITLE
fix #79494 touhouyaruosure.com

### LINKED ISSUE #79494: touhouyaruosure.com
### Issue URL (Ads)
 [http://touhouyaruosure.com/blog-entry-11000.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Ftouhouyaruosure.com%2Fblog-entry-11000.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Japanese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#79494

Should we move i-mobile.co.jp to japanese adservers as `||i-mobile.co.jp^$third-party`?

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -18,6 +18,11 @@ gigafile.nu###shinkarionz
 gigafile.nu###geniee_overlay_outer
 altema.jp,gorillawiki.jp##.new_koukokuwaku_wrapper
 mainichi.jp###index-remarkable
+touhouyaruosure.com##div[style^="margin:0;padding:5px;"]
+touhouyaruosure.com##a[href^="https://affiliate.suruga-ya.jp"]
+touhouyaruosure.com##a[href^="https://www.dlsite.com/"][target="_blank"]
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:23]: `touhouyaruosure.com##a[href^="https://www.dlsite.com/"][target="_blank"]` is it necessary to use `[target="_blank"]`?

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -18,6 +18,11 @@ gigafile.nu###shinkarionz
...
+||i-mobile.co.jp/script/ads.js
+||i-mobile.co.jp/script/adsnativepc.js
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:25]: We have `||i-mobile.co.jp^` in the AdGuard Japanese.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
@Yuki2718 please also review this pull request.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:23]: Can you check `touhouyaruosure.com##.footer ~ div.main a[href^="https://"][rel]` instead of those rules?

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:23]: @zloyden There's a link without [rel] at `http://touhouyaruosure.com/blog-entry-9000.html`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `||getchu.com/api/item_af.phtml/` as suggested - found on `http://touhouyaruosure.com/blog-entry-9000.html` as in the comment line.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:23]: @Yuki2718 Ok, I found it. What about these rules?
```
touhouyaruosure.com##.footer ~ div.main a[href^="https://"]:not([href*="twitter.com/"])
touhouyaruosure.com##.main > div[style^="margin:0;padding:5px;"]
```

**Comment by zloyden** [comment]
> Add `||getchu.com/api/item_af.phtml/` as suggested - found on `http://touhouyaruosure.com/blog-entry-9000.html` as in the comment line.

Also add this for app `touhouyaruosure.com##iframe[src^="https://image.getchu.com/api/item_af.phtml/"]`.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:23]: Yup, working fine.

**Comment by Yuki2718** [comment]
> Also add this for app `touhouyaruosure.com##iframe[src^="https://image.getchu.com/api/item_af.phtml/"]` without a hint.

Is this still needed? Won't the element be removed from the source on app?

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply JapaneseFilter/sections/specific.txt:23]: Ok, then @stanislav-atr you can exchange your rules to the above ones.

**Comment by zloyden** [comment]
> Is this still needed? Won't the element be removed from the source on app?

Generally, we need such rules, because app doesn't hide it via request.
Strange, that in latest nightly it works as on extension, but let's keep this rule for now.

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
@zloyden Thx, I asked because it could potentially mean I should revise past commits. Good to know it'll be addressed in coming versions.

**Comment by Yuki2718** [comment]
@stanislav-atr Use non rel version: `touhouyaruosure.com##.footer ~ div.main a[href^="https://"]:not([href*="twitter.com/"])`

File: JapaneseFilter/sections/specific.txt [hunk 3]
```diff
@@ -18,6 +18,10 @@ gigafile.nu###shinkarionz
 gigafile.nu###geniee_overlay_outer
 altema.jp,gorillawiki.jp##.new_koukokuwaku_wrapper
 mainichi.jp###index-remarkable
+touhouyaruosure.com##.footer ~ div.main a[href^="https://"][rel]
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:21]: Please exchange this rule to `touhouyaruosure.com##.footer ~ div.main a[href^="https://"]:not([href*="twitter.com/"])`

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Review by Yuki2718 — APPROVED** [review]

**Pull request merged**